static/app/views/starfish/utils/useSpansQuery.tsx

```diff
@@ -55,6 +71,35 @@ function useWrappedQuery(queryString: string, initialData?: any) {
   return {isLoading, data};
 }
 
+function useWrappedDiscoverTimeseriesQuery(eventView: EventView, initialData?: any) {
```

Comment:
Nice! I just made a similar function for this, but I'll work off this one as it's in the shared utils.